sdk/cognitiveservices/cognitiveservices-luis-authoring/README.md

```diff
@@ -22,83 +17,81 @@ npm install @azure/cognitiveservices-luis-authoring
 
 #### nodejs - Authentication, client creation and listPhraseLists features as an example written in TypeScript.
 
-##### Install @azure/ms-rest-azure-js
+##### Install @azure/ms-rest-nodeauth
```

Comment:
This change should not have happened. The cognitive services packages do not support AAD auth. Same feedback for the browser sample

@sarangan12 I assumed that this change happens only if the ARM flag is passed to the code gen. Can you share what @mariam606 can do to avoid this change?

Or did we always manually update the readme file after generating the code?